cmd/containerboot/main.go

```diff
@@ -102,19 +105,16 @@ func main() {
 		Root:            defaultEnv("TS_TEST_ONLY_ROOT", "/"),
 	}
 
-	if cfg.ProxyTo != "" && cfg.UserspaceMode {
+	if cfg.IngressProxyTo != "" && cfg.UserspaceMode {
```

Comment:
You need to add a similar check for egress

Reply:
We actually already do it in an init container as well https://github.com/tailscale/tailscale/blob/main/cmd/k8s-operator/manifests/proxy.yaml#L13-L21. I'm guessing we can then remove the init container - the check in the code is smarter as it allows for cases where it is not allowed to run sysctl commands

Reply:
the init container isn't for checking, its to ensure we have ip forwarding enabled.

Reply:
the code also enables, it writes `1` here if not already https://github.com/tailscale/tailscale/blob/main/cmd/containerboot/main.go#L607

Reply:
(I did add the check to code though)